extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/photos/GooglePhotosImporter.java

```diff
@@ -218,11 +218,11 @@ long importPhotoBatch(
         Pair<InputStream, Long> inputStreamBytesPair =
             getInputStreamForUrl(jobId, photo.getFetchableUrl(), photo.isInTempStore());
 
-        try (InputStream s = inputStreamBytesPair.getFirst()) {
+        try (InputStream s = inputStreamBytesPair.getLeft()) {
           String uploadToken = getOrCreatePhotosInterface(jobId, authData).uploadPhotoContent(s);
           mediaItems.add(new NewMediaItem(cleanDescription(photo.getDescription()), uploadToken));
           uploadTokenToDataId.put(uploadToken, photo);
-          uploadTokenToLength.put(uploadToken, inputStreamBytesPair.getSecond());
+          uploadTokenToLength.put(uploadToken, inputStreamBytesPair.getRight());
         }
```

Comment:
Is there a reason we don't use `InputStreamWrapper` directly instead of working with pairs?

```
InputStreamWrapper streamWrapper = jobStore.getStream(jobId, fetchableUrl);
try (InputStream s = streamWrapper.getStream()) {
  ....
  uploadTokenToLength.put(uploadToken, inputStreamBytesPair.getBytes());
}
```

It would also allow us to get rid of `inputStreamBytesPair` method.

Reply:
Please disregard my review, I totally missed the fact `getInputStreamForUrl` does more than just creating a stream wrapper. Sorry